CMakeLists.txt

```diff
@@ -1,11 +1,25 @@
-cmake_minimum_required(VERSION 3.27)
+cmake_minimum_required(VERSION 3.26)
 project(clings C)
 
 set(CMAKE_C_STANDARD 11)
 
+find_package(yaml CONFIG REQUIRED)
+
+include_directories(src)
+
 add_executable(clings main.c
         src/execution/execution.c
+        src/execution/execution.h
+        src/types/types.h
```

Comment:
I thought you wanted to remove the .h from here ?

Reply:
à un oubli